src/test/java/com/galvanize/customer/services/CustomerServiceTests.java

```diff
@@ -17,28 +23,51 @@
     @Autowired
     CustomerService service;
 
+    Customer c;
+
     @Before
     public void setUp() throws Exception {
-
+        c = new Customer();
+        c.setCustomerId(312L);
+        c.setFirstName("Jim");
+        c.setLastName("Jones");
+        c.setAddress("111 Main St");
+        c.setCity("Dallas");
+        c.setState("TX");
+        c.setZip("90210");
+        c.setPhoneNumber("111-222-3333");
```

Comment:
Missing the save step.  ie: c = repository.save(c);  The attribute "c" will be assigned to the Customer that is returned by the save() method.  NOTE: you will need to autowire repository in the class.